serving/src/main/java/ai/djl/serving/http/AdapterManagementRequestHandler.java

```diff
@@ -37,9 +46,14 @@
 /** A class handling inbound HTTP requests to the management API for adapters. */
 public class AdapterManagementRequestHandler extends HttpRequestHandler {
 
+    private static final Logger logger =
+            LoggerFactory.getLogger(AdapterManagementRequestHandler.class);
+
     static final Pattern ADAPTERS_PATTERN =
             Pattern.compile("^(/models/[^/^?]+)?/adapters([/?].*)?");
 
+    private final ConfigManager config = ConfigManager.getInstance();
```

Comment:
we shouldn't make this an object property, let's just fetch the instance directly in `OnException` since that's the only place we use it

Reply:
Changed.